spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java

```diff
@@ -270,6 +271,9 @@ public boolean deRegisterInstance(String serviceId, String instanceId)
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         LOGGER.info(response.getContent());
         Microservice result = objectMapper.readValue(response.getContent(), Microservice.class);
+        if (result == null || result.getInstances() == null) {
+          return instanceList;
```

Comment:
I think this line should add logs before return statement for exception branch.

Reply:
done